pkg/kubelet/kubelet_test.go

```diff
@@ -340,7 +340,8 @@ func newTestKubeletWithImageList(
 		kubelet.getPodsDir(),
 		kubelet.recorder,
 		false, /* experimentalCheckNodeCapabilitiesBeforeMount*/
-		false /* keepTerminatedPodVolumes */)
+		false, /* keepTerminatedPodVolumes */
+		30*time.Second)
```

Comment:
nit: To improve variability, for any static value, put a small `/* ... */` comment indicating what the parameter name is.